### ISSUE TITLE
Daily builds not published any more in the last months to MyGet

### ISSUE THREAD

**Issue by bergmeister** [PowerShell/PowerShell#7955]
Steps to reproduce
------------------

Run the install-powershell.ps1 script [here](https://github.com/PowerShell/PowerShell/blob/master/tools/install-powershell.ps1) with the `-Daily` switch (`.\tools\install-powershell.ps1 -Daily`) or just `iex "& { $(irm 'https://aka.ms/install-pscore' ) } -Daily"` for the faint-hearted ;-)

Expected behavior
-----------------

Downloaded PowerShell version from [MyGet]("https://powershell.myget.org/F/powershell-core-daily") is [6.1-preview831](https://ci.appveyor.com/project/PowerShell/powershell-f975h/build/v6.1.0-rc.831) or higher

Actual behavior
---------------

Downloaded PowerShell version from [MyGet]("https://powershell.myget.org/F/powershell-core-daily") is stuck at 6.1.0-preview-761

Environment data
----------------

```powershell
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      6.1.0
PSEdition                      Core
GitCommitId                    6.1.0
OS                             Microsoft Windows 10.0.17134
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by SteveL-MSFT** [comment]
cc @adityapatwardhan

**Comment by adityapatwardhan** [comment]
I will have a look at this.

**Comment by adityapatwardhan** [comment]
This should be fixed now.

```PowerShell
PS> iex "& { $(irm 'https://aka.ms/install-pscore' ) } -Daily"
VERBOSE: Destination: C:\Users\adityap\AppData\Local\Microsoft\powershell-daily
VERBOSE: Daily package found. Name: powershell-win-x64-win7-x64; Version: 6.1.0-rc-837
VERBOSE: Change icon to disambiguate it from a released installation
```

**Comment by bergmeister** [comment]
Thanks, I can confirm the fix :)

**Comment by bergmeister** [comment]
@adityapatwardhan Just a reminder that the daily build has not been updated since February and is still on `6.2.0-dailypreview4.12015`. I am running `.\tools\install-powershell.ps1 -Daily` from the main `master` branch

**Comment by adityapatwardhan** [comment]
I will have a look at it this week.

**Comment by adityapatwardhan** [comment]
@bergmeister - Can you give this a try:

```PowerShell
Install-Package powershell-win-x64-win7-x64 -Version 6.2.0-dailyrc1-19313 -Source https://powershell.myget.org/F/powershell-core-daily/api/v3/index.json
```

**Comment by bergmeister** [comment]
Thanks, it is fixed now, locally I get the `6.2.0-dailyrc1.19498` now running the `.\tools\install-powershell.ps1 -Daily` script :)
